syndesis/plugins/org.fusesource.ide.syndesis.extension.ui/src/org/fusesource/ide/syndesis/extensions/ui/wizards/pages/SyndesisExtensionProjectWizardVersionsPage.java

```diff
@@ -0,0 +1,211 @@
+	 */
+	@Override
+	public void createControl(Composite parent) {
+		Composite container = new Composite(parent, SWT.NULL);
+		container.setLayout(new GridLayout(3, false));
+
+		Label springBootVersionLabel = new Label(container, SWT.NONE);
+		springBootVersionLabel.setText(Messages.newProjectWizardExtensionVersionsPageSpringBootVersionLabel);
+		GridData gridData = new GridData(SWT.FILL, SWT.TOP, true, false, 2, 1);
+		springBootVersionCombo = new Combo(container, SWT.BORDER | SWT.DROP_DOWN);
+		springBootVersionCombo.setLayoutData(gridData);
+		springBootVersionCombo.setToolTipText(Messages.newProjectWizardExtensionVersionsPageSpringBootVersionTooltip);
+		fillSpringBootVersions();
+		springBootVersionCombo.addSelectionListener(selectionListener);
+		springBootVersionCombo.addModifyListener( (ModifyEvent e) -> validate() );
+		
+		Label camelVersionLabel = new Label(container, SWT.NONE);
+		camelVersionLabel.setText(Messages.newProjectWizardExtensionVersionsPageCamelVersionLabel);
+		gridData = new GridData(SWT.FILL, SWT.TOP, true, false, 1, 1);
+		camelVersionCombo = new Combo(container, SWT.BORDER | SWT.DROP_DOWN);
+		camelVersionCombo.setLayoutData(gridData);
+		camelVersionCombo.setToolTipText(Messages.newProjectWizardExtensionVersionsPageCamelVersionTooltip);
+		fillCamelVersions();
+		camelVersionCombo.addSelectionListener(selectionListener);
+		camelVersionCombo.addModifyListener( (ModifyEvent e) -> validate() );
+		
+		Button camelVersionValidationBtn = new Button(container, SWT.PUSH);
+		GridData camelButtonData = new GridData(SWT.RIGHT, SWT.CENTER, false, false, 1, 1);
+		camelVersionValidationBtn.setLayoutData(camelButtonData);
+		camelVersionValidationBtn.setText(Messages.newProjectWizardExtensionVersionsPageCamelVersionValidationLabel);
+		camelVersionValidationBtn.setToolTipText(Messages.newProjectWizardExtensionVersionsPageCamelVersionValidationTooltip);
+		camelVersionValidationBtn.addSelectionListener(new VersionValidationHandler());
+		
+		Label syndesisVersionLabel = new Label(container, SWT.NONE);
+		syndesisVersionLabel.setText(Messages.newProjectWizardExtensionVersionsPageSyndesisVersionLabel);
+		gridData = new GridData(SWT.FILL, SWT.TOP, true, false, 2, 1);
+		syndesisVersionCombo = new Combo(container, SWT.BORDER | SWT.DROP_DOWN);
+		syndesisVersionCombo.setLayoutData(gridData);
+		syndesisVersionCombo.setToolTipText(Messages.newProjectWizardExtensionVersionsPageSyndesisVersionTooltip);
+		fillSyndesisVersions();
+		syndesisVersionCombo.addSelectionListener(selectionListener);
+		syndesisVersionCombo.addModifyListener( (ModifyEvent e) -> validate() );
+		
+		setControl(container);
+		
+		springBootVersionCombo.setFocus();
+		validate();
+	}
+
+	private void fillSpringBootVersions() {
+		// for the moment we only support 1.5.8
+		springBootVersionCombo.add("1.5.8.RELEASE");
+		springBootVersionCombo.select(0);
+	}
+	
+	private void fillCamelVersions() {
```

Comment:
There is already an enhancement request open that Ignite will provide an API to query for those versions. Whenever thats available we can change that page.

Reply:
we don't know when it will be available and when we will have time to leverage it. That's why i'm suggesting to remove or set as read-only the field for this iteration.

Reply:
will be FUSETOOLS-2696 and https://github.com/syndesisio/syndesis/issues/636